tests/tests/test_deployment.py

```diff
@@ -196,6 +196,67 @@ def test_deployments_new_no_artifact(self):
         else:
             raise AssertionError("expected to fail")
 
+    def test_deplyments_get_devices(self):
+        """Create deployments, get devices with pagination"""
+        devices = []
+        devices_qty = 30
+        device_ids = []
+        default_per_page = 20
+        device_type = "test-hammer-type"
+
+        # create devices
+        for i in range(0, devices_qty):
+            device = Device(device_type)
+            self.inventory_add_dev(device)
+            devices.append(device)
+            device_ids.append(device.devid)
+
+        data = b"foo_bar"
+        artifact_name = "pagination-test-" + str(uuid4())
+        # come up with an artifact
+        with artifact_from_data(
+                name=artifact_name, data=data, devicetype=device_type
+        ) as art:
+            ac = SimpleArtifactsClient()
+            ac.add_artifact(
+                description="some description", size=art.size, data=art
+            )
+
+            new_dep = self.d.make_new_deployment(
+                name="pagination deployment", artifact_name=artifact_name, devices=device_ids
+            )
+            dep_id = self.d.add_deployment(new_dep)
+
+            for dev in devices:
+                dc = SimpleDeviceClient()
+                dc.get_next_deployment(
+                    dev.fake_token,
+                    artifact_name="different {}".format(artifact_name),
+                    device_type=dev.device_type,
+                )
+
+            # check default 'page' and 'per_page' values
+            res = self.d.client.Management_API.Devices_List_in_Deployment(
```

Comment:
If you change the operation ID in the API docs, you'll need to adjust this method name as well